spec/examples/actions/ui.spec.js

```diff
@@ -55,4 +60,30 @@ describe('interfaceStateActions', () => {
       )
     );
   });
+
+  describe('userRequestedFocusedLine', () => {
+    beforeEach(
+      () => store.dispatch(userRequestedFocusedLine('javascript', 4, 2))
+    );
+
+    it('sets requestedFocusedLine to given value', () => {
+      assert.deepEqual(
+        store.getState().ui.getIn(['editors', 'requestedFocusedLine']).toJS(),
+        {language: 'javascript', line: 4, column: 2}
+      );
+    });
+  });
+
+  describe('userRequestedFocusedLine', () => {
```

Comment:
Is this correct? should this describe editorFocusedRequestedLine, and the second describe say that it nulls out the focused line state?